lib/config/rules/typescript.js

```diff
@@ -79,7 +75,13 @@ module.exports = {
   // Disallow the declaration of empty interfaces. (no-empty-interface from TSLint)
   '@typescript-eslint/no-empty-interface': 'off',
   // Requires using either T[] or Array<T> for arrays (array-type)
-  '@typescript-eslint/array-type': ['error', 'array'],
+  '@typescript-eslint/array-type': [
+    'error',
+    {
+      default: 'array-simple',
+      readonly: 'generic',
```

Comment:
I don't think we can use anything other than `generic` for the `readonly` property until we are on typescript 3.4 or greater.

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html